lib/cheep.rb

```diff
@@ -0,0 +1,24 @@
+class Cheep
+
+  attr_reader :post, :id
+
+  def initialize(post:, id:)
```

Comment:
Looking at your earlier commit, it's interesting to see your process of writing code and then later extracting a class when you realised it was needed.